### ISSUE TITLE
Some examples missing graphics resources

### ISSUE THREAD

**Issue by eliotb** [ppb/pursuedpybear#479]
AFAICS only animated_sprites and loading_screen have a resource subdirectory

getting_started_sample and others are missing the required graphics, and display squares instead.

**Comment by pathunstrom** [comment]
How are you running them?

getting_started_sample is just the code for https://ppb.readthedocs.io/en/master/getting-started.html you would still need to add your own images to the code. It should be squares until you add your own images.

**Comment by pathunstrom** [comment]
Decided to take a survey of the collected samples:

* animated_sprites: Has my blob animation
* external_event_loop_integration: has resources without a resources folder
* frame_count: doesn't need resources, should open screen and print to the terminal
* getting-started-sample: Just the code from the tutorial, no art, should render squares
* keyboard_and_mouse_control: currently bugged( #480), but has images and sound effects
* loading_screen: lots of resources
* two-phase-updates: no resources, uses the default squares

Does this match what you're seeing?

**Comment by eliotb** [comment]
Yes.  It is not clear that the images are meant to be missing, when the examples emit warnings about files not found.

``` 
$ python getting-started-sample/main.py 
WARNING:ppb.assetlib:File not found: 'target.png'
WARNING:ppb.assetlib:File not found: 'player.png'
```

```
 $ python two-phase-updates/three_body.py 
WARNING:ppb.assetlib:File not found: 'planet.png'
```

**Comment by pathunstrom** [comment]
Hey, @eliotb I made separate issues, can you look at #481 and #482 and let me know if those would be enough to reduce the confusion? If you have other ideas, share them here and we can figure it out.

**Comment by pathunstrom** [comment]
This hasn't had further discussion, I'm going to operate on the assumption that the more specific tickets will handle the issues raised here. Closing this in the meantime.
